agent/tool-scripts/jmap

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 8; sh-indentation: 8; sh-indent-for-case-alt: + -*-
+# The following script collect heap used stats for jvm and containerized jvm
+
+script_path=`dirname $0`
+script_name=`basename $0`
+pbench_bin="`cd ${script_path}/..; /bin/pwd`"
+
+# source the base script
+. "$pbench_bin"/base
+
+# Perftool scripts must provide the following functions
+# 1) Install the tool
+# 2) Start data collection
+# 3) Stop data collection
+# 4) post-process the data
+
+# Defaults
+tool=jmap
+tool_bin=/usr/bin/$tool
+group=default
+dir=""
+mode=""
+iteration="1"
+options="none"
+jcn=""
+
+function usage() {
+echo "
+USAGE:
+	$script_name: you specified an invalid option
```

Comment:
Can we fix up the formatting of this help usage to be like what we do in other tools?

Reply:
yep